code/modules/clothing/spacesuits/chronosuit.dm

```diff
@@ -274,7 +274,7 @@
 	if(holder)
 		if(user == holder)
 			if(loc == user)
-				loc = get_turf(user)
+				forceMove(get_turf(user))
```

Comment:
While you're messing with stuff here could you reduce indentation/help readability a bit.
```
if(!holder)
   qdel(src)
if(user == holder)
   ...
```